kalite/knowledgemap/templates/knowledgemap/knowledgemap.html

```diff
@@ -0,0 +1,616 @@
+{% extends base_template %}
+{% load i18n %}
+{% load staticfiles %}
+
+{% block practice_active %}active{% endblock practice_active %}
+
+{% block headcss %}{{ block.super }}
+    <link rel='stylesheet' type='text/css' href='{% static "css/distributed/tipsy.css" %}' />
+    <link rel='stylesheet' type='text/css' href='{% static "css/knowledgemap/knowledgemap.css" %}' />
+{% endblock headcss %}
+
+{% block headjs %}{{ block.super }}
+    <script src="{% static 'js/distributed/jquery.tipsy.js' %}"></script>
+
+    <script type="text/javascript">
+        var TOPIC_DATA_URLS = {
+            Domains: '{{ data_url }}/map_domains.json',
+            Subjects: '{{ data_url }}/map_subjects.json',
+            Topics: '{{ data_url }}/topics.json',
+            Tutorials: '{{ data_url }}/map_tutorials.json',
+            Exercises: '{{ data_url }}/exercises.json'
+        }
+        var ALL_TOPICS_URL = '{{ topics_url }}';
+    </script>
+{% endblock headjs %}
+
+{% block content %}
+<script src="http://d3js.org/d3.v3.min.js"></script>
```

Comment:
This won't work offline. We already have d3 locally, check that it is an up to date version and change it if necessary.